sale_report_supplier_filter/reports/report_all_channels_sales.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ReportAllChannelsSales(models.Model):
+    _inherit = "report.all.channels.sales"
+
+    seller_id = fields.Many2one(
+        related='product_id.seller_ids.name',
```

Comment:
IMHO, you have to say in README that this is applicable for main supplier at time of report, if the main supplier was another one at sale time, the result could be incorrect. Or I misunderstand the aim of this?

Reply:
Hello @rousseldenis , it shows of **any** supplier , nor the main. It is intended and discussed with different customers. All customer ask for the same thing, **any**.